code/__DEFINES/chemistry.dm

```diff
@@ -2,8 +2,6 @@
  * Chemistry defines
  */
 
-/// Amount of bottle icon variations in total
-#define BOTTLE_ICON_CHOICES 4
```

Comment:
Why did you remove what I just suggested to be added

Reply:
I think fixed